starcheck/src/starcheck.pl

```diff
@@ -137,7 +142,7 @@
     # time to be a time run_start_time days back from backstop start
     try:
         run_start_time = float(run_start_time)
-    except ValueError:
+    except (TypeError, ValueError):
```

Comment:
This is really a bugfix on #329, so I'm not sure if I should just put it here or in a new PR.

Reply:
Fine here.  Did this come up with `None`?

Reply:
Yes.  I probably should add a comment.  And I forgot that you might not git a github notification when I just commented on the other commit https://github.com/sot/starcheck/commit/dca234ef29dd98ed0cdf6bcae1bb9cce17317785#commitcomment-34330977

Reply:
I added a comment.